.github/workflows/shinyapps_deploy.yml

```diff
@@ -0,0 +1,70 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/master/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+
+name: shiny-apps-deploy
+
+on:
+  push:
+    branches:
+      - master
+      - shinyapps-and-docker
+      - release*
+
+jobs:
+  shiny-deploy:
+    runs-on: ubuntu-latest
+ 
+    steps:
+ 
+      - name: Install System Dependencies
+        run: |
+          sudo apt-get update
+          # The rsconnect R package needs cUrl.  This app needs the Java JDK (runtime AND compiler).
+          sudo apt-get install -y libcurl4-openssl-dev default-jdk
+ 
+      - uses: actions/checkout@v2
+        
+      # We don't use renv, so we want to skip the "renv::activate()" command in .Rprofile
+      - name: disable renv
+        run: rm .Rprofile
+       
+      - name: Install R packages
+        run: |
+          Sys.setenv(GITHUB_PAT="${{ secrets.pat_for_shinyapps_action }}")
+          source("install-pkgs.R")
+        shell: sudo Rscript {0}
+       
+      - name: Deploy app
+        run: |
+          branch<-Sys.getenv("GITHUB_REF_NAME")
+          repo<-Sys.getenv("GITHUB_REPOSITORY")
+          appName<-strsplit(repo, "/")[[1]][2]
+          if (!startsWith(branch, "release")) {
+             appName = paste(appName, "staging", sep="-")
+          }
```

Comment:
@j-hendrickson-sage Note that this is set up so that we will need to start creating release branches in order to deploy to production. Let's sync up to talk about our gitflow and versioning before you try to deploy to prod.

Reply:
Elsewhere we deploy based on a *tag* rather than a *branch*, which is totally legit'.  You can feel free to do it either way.

Reply:
@JessterB Let's sync tomorrow; I will hold off approval until I get more time to inspect the changes.